src/components/ConfigOptionItem.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react"
+
+type ConfigOptionItemProps = {
+    Required: boolean
+    FullPath: string
+    Description: string
+    Example?: string
+}
+
+export const ConfigOptionItem: React.FunctionComponent<ConfigOptionItemProps> = ({
```

Comment:
Based on your emoji in slack I think you already saw this, but we have an existing pattern of `HclListItem` blocks.  That said, you're documenting yaml here, not HCL, so maybe a new component still makes sense?  Just want to make sure you at least considered the existing components as options